onnx_mxnet/backend.py

```diff
@@ -0,0 +1,98 @@
+# Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+# coding: utf-8
+from .import_onnx import GraphProto
+import mxnet as mx
+import numpy as np
+from onnx.backend.base import Backend, BackendRep
+from collections import namedtuple
+
+"""Using these functions for onnx test infrastructure."""
```

Comment:
Can you elaborate more in the comment about the onnx test infra, and what this class encapsulates?